sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/MiscBenchmark.scala

```diff
@@ -227,4 +228,45 @@ class MiscBenchmark extends BenchmarkBase {
     generate stack wholestage on                   836 /  847         20.1          49.8      15.5X
      */
   }
+
+  ignore("generate explode big struct array") {
+    val N = 60000
+
+    val spark = sparkSession
+    import org.apache.spark.sql.functions._
+    import spark.implicits._
+
+    val df = sparkSession.sparkContext.parallelize(Seq(("1",
+      Array.fill(N)({
+        val i = math.random
+        (i.toString, (i + 1).toString, (i + 2).toString, (i + 3).toString)
+      })))).toDF("col", "arr")
+
+    runBenchmark("generate big struct array", N) {
+      df.withColumn("arr_col", explode('arr)).select("col", "arr_col.*").count
+    }
+
+  }
+
+    /*
```

Comment:
This benchmark result should be moved up to be included in the ignored test block.